theme_odoo_experts/views/snippets/s_references.xml

```diff
@@ -4,11 +4,12 @@
 <template id="s_references" inherit_id="website.s_references">
     <!-- Section -->
     <xpath expr="//section" position="attributes">
-        <attribute name="class" add="pt0 pb56 o_cc o_cc2" remove="pt48 pb32" separator=" "/>
+        <attribute name="class" add="pt0 pb56 o_cc2" remove="pt80 pb80 o_cc1" separator=" "/>
     </xpath>
-    <!-- Remove title and subtitle -->
+    <!-- Remove title, subtitle and button -->
     <xpath expr="//h2" position="replace"/>
     <xpath expr="//p" position="replace"/>
+    <xpath expr="//a" position="replace"/>
```

Comment:
This theme has the same issue
![Capture d’écran 2024-08-05 à 08 36 38](https://github.com/user-attachments/assets/64ee946e-7d53-458d-b0e3-76d5b7e82a20)